### DISCUSSION TITLE
ARM

### DISCUSSION THREAD

**Discussion by Shuzhengz** [JakeRoggenbuck/auto-clock-speed#427] [Ideas]
We should probably decided if we are going to support ARM devices, because if not then we want to update our README and wiki

**Comment by JakeRoggenbuck** [comment]
Yea

**Comment by JakeRoggenbuck** [comment]
We could just ask if anyone would like it.

**Comment by JakeRoggenbuck** [comment]
I know that one person uses or used acs on a raspberry pi.

**Comment by Shuzhengz** [comment]
I tink what we could do is say that ARM devices should work, but they are not officially supported as we cannot test them

**Comment by Shuzhengz** [comment]
Also do you know if big.LITTLE architecture devices running linux have any different governors? Their cores and power controls are very wack

**Comment by JakeRoggenbuck** [comment]
> I tink what we could do is say that ARM devices should work, but they are not officially supported as we cannot test them

Yea, I am good with that, and we could ask community members that if they test it, they can let us know and we could post the findings

**Comment by JakeRoggenbuck** [comment]
> Also do you know if big.LITTLE architecture devices running linux have any different governors? Their cores and power controls are very wack

not sure, what would be good to test on?

**Comment by Shuzhengz** [comment]
> > Also do you know if big.LITTLE architecture devices running linux have any different governors? Their cores and power controls are very wack
> 
> not sure, what would be good to test on?

any arm v8 or v9 SoC, or cortex A75 and A55 architectures
also any intel 12th gen mobile CPU (12xxx U, H, G, M, or L), as well as nvidia tegra 3 and above
and apple A10 or above, M1 or M2, if you can get them to work ofc
also a lot more, but they're too much to list so yea lol
